backend/apid/routers/authentication.go

```diff
@@ -124,128 +102,118 @@ func (a *AuthenticationRouter) logout(w http.ResponseWriter, r *http.Request) {
 
 	// Get the access token claims
 	if value := r.Context().Value(types.AccessTokenClaims); value != nil {
-		accessClaims = value.(*types.Claims)
+		accessClaims = value.(*v2.Claims)
 	} else {
-		http.Error(w, "could not retrieve the access token claims", http.StatusBadRequest)
+		http.Error(w, "invalid access token", http.StatusBadRequest)
 		return
 	}
 
 	// Get the refresh token claims
 	if value := r.Context().Value(types.RefreshTokenClaims); value != nil {
-		refreshClaims = value.(*types.Claims)
+		refreshClaims = value.(*v2.Claims)
 	} else {
-		http.Error(w, "could not retrieve the refresh token claims", http.StatusBadRequest)
+		http.Error(w, "invalid refresh token", http.StatusBadRequest)
 		return
 	}
 
 	// Remove the access & refresh tokens from the access list
-	tokensToRemove := []string{accessClaims.Id, refreshClaims.Id}
-	if err := a.store.DeleteTokens(refreshClaims.Subject, tokensToRemove); err != nil {
-		err = fmt.Errorf("could not remove the access and refresh tokens from the access list: %s", err)
-		logger.WithField("user", refreshClaims.Subject).Error(err)
-		http.Error(w, err.Error(), http.StatusInternalServerError)
+	if err := a.store.RevokeTokens(accessClaims, refreshClaims); err != nil {
+		logger.WithField("user", refreshClaims.Subject).Errorf(
```

Comment:
Same thing here, can we log the error `WithError(err)` or does that have sensitive info as well?